packages-backend/loggers/src/env.ts

```diff
@@ -0,0 +1,6 @@
+const env = process.env.NODE_ENV;
+const isDev = !env || env === 'development';
+const isProd = env === 'production';
+const isTest = env === 'test';
+
+export { isDev, isProd, isTest };
```

Comment:
I would also have this as an arg to the middlewars. `envGetters` maybe. Not 100% also just making notes for myself.